src/Elmish.WPF.Tests/StaticViewModelTests.fs

```diff
@@ -0,0 +1,347 @@
+module StaticViewModelTests.M
+
+open System
+open System.Collections.Concurrent
+open System.Collections.ObjectModel
```

Comment:
```suggestion
open System.Collections.Concurrent
```

Reply:
I'd rather leave these here for now, same reason as above.